### PR TITLE
add legends to named map vizjson

### PR DESCRIPTION
**Pull request by gfiorav**
Forgot to add it here.

Please have a look @juanignaciosl !

### CODE REVIEW

File: app/controllers/carto/api/vizjson3_presenter.rb [hunk 1]
```diff
@@ -269,6 +269,8 @@ def data_for_carto_layer(layer_vizjson)
           data[:legend] = legend
         end
 
+        data[:legend] = layer_vizjson[:legends] || []
```
Comment (juanignaciosl) [inline app/controllers/carto/api/vizjson3_presenter.rb]: This will always override previous `legend` assignment, which is  hardly what you want.

Comment (gfiorav) [inline app/controllers/carto/api/vizjson3_presenter.rb]: oh! typo! it should read legends.

ONE LINE AND IT HAS A TYPO

sorry, changing

**Comment by gfiorav** [comment]
Sorry for the careless typo, see now please

**Comment by juanignaciosl** [comment]
LGTM 👍

**Comment by gfiorav** [comment]
@juanignaciosl you have to approve the review! 🙈

**Comment by juanignaciosl** [comment]
Sorry!

**Pull request merged**